docker-compose.yaml

```diff
@@ -25,6 +26,7 @@ services:
       - 9966:9966
       - 9967:9967
     environment:
+      CASSANDRA_USE_ASTRA: "false"
```

Comment:
I tried to keep all of our edits in the `kustomization.yaml`, however, I found it useful to run the local docker-compose setup. This was the minimum change needed to make docker-compose work without the cloud db they advertise.